### PR TITLE
commit model.py

### PR DESCRIPTION
**Pull request by yjchoi1**

### CODE REVIEW

File: model.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    ]
+
+    # compile
+    model.compile(loss=loss,
+                  optimizer=optimizers.Adam(learning_rate=learning_rate),  # default learning_rate is 0.001
+                  metrics=metric)
+
+    # fit the model
+    history = model.fit(train_x, train_y,
+                        epochs=10, batch_size=32, verbose=2,
```
Comment (kks32) [inline model.py:44]: Why don't you make `epoch`, `batch_size`, and `validation_split` all JSON properties for learning and you can pass it as a parameter to the `compile_fit` function. Keep the number of function arguments to a minimum.

File: train.py [hunk 2]
```diff
@@ -30,3 +31,38 @@
 rnn_data_test = preparedata.rnn_inputs(
     dfs=normalized_test_dfs, features=input["features"], targets=input["targets"], window_length=input["window_length"]
 )
+
+# %% set variables for train and test
+
+# train
+train_x_rnns = rnn_data_train["x_rnns"]  # variables for plotting
```
Comment (kks32) [inline train.py:42]: How are you splitting the training and validation data?

**Review by kks32 — CHANGES_REQUESTED** [review]
Looking good, some changes to the function parameters for the model.

File: train.py (hunk 2 above)
Comment (yjchoi1) [reply train.py:42]: I use tf.model.fit method to split train and validation. I set it randomly select 20% of my training data as the validation set.

**Review by kks32 — APPROVED** [review]

**Review by kks32 — APPROVED** [review]
